functions.php

```diff
@@ -0,0 +1,71 @@
+<?php
```

Comment:
let's rename this file to `console_helpers.php` and move it to the `libraries` folder